dali/base/dasds.cpp

```diff
@@ -4091,12 +4088,12 @@ bool translateOldFormat(CServerRemoteTree *parentServerTree, IPropertyTree *pare
 
 void CSDSTransactionServer::processMessage(CMessageBuffer &mb)
 {
-    // 1) Decrement sds_pending_requests here.
+    // 1) Increment sds_requests.started here.
     // NB: request has started, no longer pending
-    // See CSDSTransactionServer::run() for where needs to be incremented. 
+    // See CSDSTransactionServer::run() for where needs to be incremented.
 
-    // 2) Ensure sds_active_requests is decremented, when processMessage() is finished
-    // e.g. use a scoped object here, and decrement in dtor.
+    pSdsRequestsStarted->inc(1);  // this is where the active count was incremented before
+    COnScopeExit incCompletedOnExit([&](){ pSdsRequestsCompleted->inc(); });
```

Comment:
@jakesmith Could the inc here also be a fastInc ?

Reply:
no, I don't see how it can, there will multiple concurrent threads at this point (in processMessage via thread pool)

Reply:
It shall remain as is.